### ISSUE TITLE
Should I still install tensorflow-cpu instead of tensorflow pakcage after 2.11 or above?

### ISSUE THREAD

**Issue by KichangKim** [microsoft/tensorflow-directml-plugin#319]
Tensorflow 2.11 release note says that both tensorflow and tensorflow-cpu will install same tensorflow-intel package. So can I use directml plugin with tensorflow package? or still need to install tensorflow-cpu package?

Thanks.

**Comment by PatriceVignola** [comment]
Yes, after tensorflow 2.11 you will be able to install the `tensorflow` package on Windows to use tensorflow-directml-plugin, but it will be the same as using `tensorflow-cpu`. Note that WSL will still need the `tensorflow-cpu` package, therefore tensorflow-cpu will stay as a required dependency for the `tensorflow-directml-plugin` for simplicity purposes.
